webpack.config.coffee

```diff
@@ -0,0 +1,43 @@
+path = require("path")
+
+file_name = (mode) ->
+  if mode == "production"
+    "alephbet.min.js"
+  else
+    "alephbet.js"
+
+module.exports = (env, argv) ->
+  target: "web"
+  node:
+    fs: "empty"
```

Comment:
Why did you add this?

Reply:
Good point :) probably unnecessary... I did a fair bit of trial and error by copying stuff from other places. I can try to remove it and hopefully it would still work.

Reply:
I didn't know what it means so I had to look at the documentation. I don't think you need it.

Reply:
It seems to be needed for webpack.test.config.coffee otherwise the tests fail